server/db/models/bug.js

```diff
@@ -25,10 +20,22 @@ module.exports = (sequelize, DataTypes) => {
   }
   bug.init(
     {
-      title: DataTypes.STRING,
-      description: DataTypes.STRING,
-      state: DataTypes.STRING,
-      priority: DataTypes.INTEGER,
+      title: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      description: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      state: {
+        type: DataTypes.STRING,
```

Comment:
Should this have a default value of 0 or whatever we want the inital value to be ('created'???)
Also this could be stored as an number as well